config/jobs/kubernetes-sigs/sig-windows/release-1.28-windows-presubmits.yaml

```
@@ -0,0 +1,53 @@
+presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-capz-windows-1.28
```

```suggestion
  - name: pull-kubernetes-e2e-capz-windows-1-27
```

to pass CI and match other jobs

---

updated